src/frontend/src/styles/main.css

```diff
@@ -284,7 +284,7 @@
 
   --rs-list-item-stack: calc(var(--vs-stack) * 1.5);
   --rs-list-parcel-bezel: var(--vs-bezel);
-  --rs-list-gutter: var(--vs-gutter);
+  --rs-list-gutter: calc(var(--vs-gutter) * 2);
```

Comment:
Horizontal spacing within list item was doubled as seen in the screenshot changes of this PR, previously this was rather narrow.

This increases the horizontal space between e.g. the icons and labels to be less cramped as seen in below example:

<img width="416" alt="image" src="https://github.com/user-attachments/assets/c192758d-b0d2-4571-8a8e-43f65c617ee3" />